### PR TITLE
Acknowledge Production as a slot - Azure App Service Manage Task

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

**Comment by vincentdass** [comment]
#5348

File: Tasks/AzureAppServiceManage/azureappservicemanage.ts [hunk 1]
```diff
@@ -71,6 +71,11 @@ async function run() {
 
         var endPoint = await azureStackUtility.initializeAzureRMEndpointData(connectedServiceName);
 
+        if(slotName && slotName.toLowerCase() === 'production') {
```
Comment (dikhakha) [inline Tasks/AzureAppServiceManage/azureappservicemanage.ts]: As discussed offline, Can you please handle similar case for Slot Swap action  where source or target slot is given as 'Production'

Comment (vincentdass) [reply Tasks/AzureAppServiceManage/azureappservicemanage.ts]: Addressed

File: Tasks/AzureAppServiceManage/azureappservicemanage.ts [hunk 2]
```diff
@@ -104,6 +109,15 @@ async function run() {
                 if (swapWithProduction) {
                     targetSlot = "production";
                 }
+
+                sourceSlot = sourceSlot.toLowerCase();
+                targetSlot = targetSlot.toLowerCase();
+
+                if(sourceSlot == 'production') {
+                    sourceSlot = targetSlot;
+                    targetSlot = 'production';
```
Comment (dikhakha) [inline Tasks/AzureAppServiceManage/azureappservicemanage.ts]: can you change 'swapWithProduction = true'.

File: Tasks/AzureAppServiceManage/azureappservicemanage.ts [hunk 3]
```diff
@@ -104,6 +109,15 @@ async function run() {
                 if (swapWithProduction) {
                     targetSlot = "production";
                 }
+
+                sourceSlot = sourceSlot.toLowerCase();
+                targetSlot = targetSlot.toLowerCase();
+
+                if(sourceSlot == 'production') {
```
Comment (Ajay-MS) [inline Tasks/AzureAppServiceManage/azureappservicemanage.ts]: "production" string is being used at multiple places, define a constant for it.

Comment (vincentdass) [reply Tasks/AzureAppServiceManage/azureappservicemanage.ts]: done

File: Tasks/AzureAppServiceManage/azureappservicemanage.ts (hunk 2 above)
Comment (vincentdass) [reply Tasks/AzureAppServiceManage/azureappservicemanage.ts]: done

**Pull request merged**